[RA2 Ch3]: Correction of linter errors (#3009)
cntt-n/CNTT: doc/ref_arch/kubernetes/chapters/chapter03.rst

Figure 3-2 shows in four steps how device plugins operate on a Kubernetes node:

- 1: During setup, the cluster administrator

>   (more in :ref:`ref_arch/kubernetes/chapters/chapter03:3.2.5.1 Operator Pattern`)

Reviewer: Since this is within the Chapter, the reference can be simplified to"  `Operator Pattern`_ " but even without that the numbering should be deleted.

The same is true for other references.

>   knows or discovers (as per :ref:`ref_arch/kubernetes/chapters/chapter03:3.2.1.5 Node Feature Discovery`) what kind of

Reviewer: Delete section number from reference

   an additional feature and still be conformant with Anuket.

..
   Editors note: The possibility to SR-IOV for DPDK is under discussion.
   Refer to software profile features

>    :doc:`ref_model/chapters/chapter05:here ` and hardware profile

Reviewer: ```suggestion
   :doc:`ref_model/chapters/chapter05:cloud infrastructure software profiles features and requirements`
   and hardware profile
```

>    features :doc:`ref_model/chapters/chapter05:here `.

Reviewer: ```suggestion
   features :doc:`ref_model/chapters/chapter05:cloud infrastructure hardware profiles features and requirements`.
```

There are no restrictions or constraints that Kubernetes places on the storage
that can be consumed by a workload, in terms of the requirements that are

> defined in RM sections :doc:`ref_model/chapters/chapter05:5.2.2` (software)

Reviewer: ```suggestion
defined in RM sections :doc:`ref_model/chapters/chapter05:Virtual Storage` (software)
```

> and :doc:`ref_model/chapters/chapter05:5.4.2` (hardware). The only point of difference is that Kubernetes does not have

Reviewer: ```suggestion
and :doc:`ref_model/chapters/chapter05:Storage Configurations` (hardware). The only point of difference
is that Kubernetes does not have
```
